packages/cli/src/lib/commands/create-action.ts

```diff
@@ -67,16 +76,34 @@ export const createActionCommand = new Command('create')
       },
       {
         type: 'input',
-        name: 'actionName',
+        name: 'actionDisplayName',
         message: 'Enter the action display name',
       },
       {
         type: 'input',
         name: 'actionDescription',
         message: 'Enter the action description',
+      },
+      {
```

Comment:
I think we should have flag to turn on the `OpenAI` otherwise we will break everyone experience and the tutorial.

Since this is in beta, and we need proper documentation for that but I will leave it for another pr, when doing the cli.